poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
This change does the opposite of what it claims. It removes the wheels rather than adding them. The wheels were added in #658.